setup.py

```diff
@@ -71,7 +71,6 @@ def get_version(package):
 
 
 SCRIPTS = glob('bin/scripts/*')
-SCRIPTS += ['control_node_serial/control_node_serial_interface']
```

Comment:
What is the reason for this ? (I can't remember)

Reply:
control_node_serial/control_node_serial_interface is a binary. In order to be able to use tox we need to be able to make a sdist, in order to have this sdist containing that binary we need it declared in the package . Previously it was declared as a script. But it's not a script, it's a binary, so I've moved it to package_data (see below)